class/defaults.yml

```
@@ -1,4 +1,41 @@
 parameters:
   openshift4_splunk_forwarder:
     =_metadata: {}
-    namespace: syn-openshift4-splunk-forwarder
+    multi_instance: true
+    namespace: openshift-logging
+
+    fluentd:
+      loglevel: info
+      replicas: 2
+
+      sharedkey: splunkforwarding
+
+      ssl:
+        enabled: false
+        cert: ""
+        key: ""
+        passphrase: ""
+
+      resources:
+        requests:
+          cpu: 100m
+          memory: 512Mi
+        limits:
+          cpu: 500m
+          memory: 1024Mi
+
+      # less used parameters
+      nodeselector: {}
+      affinity: {}
+      tolerations: []
+
+    splunk:
+      token: 4a8a737d-5452-426c-a6f7-106dca4e813f # default token
```

Secrets **MUST** be Vault references, and never directly included in the component parameters or the Project Syn hierarchy:

```suggestion
      token: ?{vaultkv:${cluster:tenant}/${cluster:name}/openshift4-splunk-forwarder/${_instance}/hec-token
```